go/libraries/doltcore/migrate/validation.go

```diff
@@ -67,6 +67,25 @@ func validateRootValue(ctx context.Context, old, new *doltdb.RootValue) error {
 			return fmt.Errorf("expected to find table %s in root value (%s)", name, h.String())
 		}
 
+		// Skip tables that haven't changed
+		op, ok, err := oldParent.GetTable(ctx, name)
+		if err != nil {
+			return err
+		}
+		if ok {
+			oldHash, err := o.HashOf()
+			if err != nil {
+				return err
+			}
+			oldParentHash, err := op.HashOf()
+			if err != nil {
+				return err
+			}
+			if oldHash.Equal(oldParentHash) {
+				continue
+			}
+		}
```

Comment:
Maybe I should check that the new parent commit and the new commit's table has the same hash?

Reply:
this is fine